docs/elasticsearch.md

```diff
@@ -48,3 +48,17 @@ Copy the mapped IP and port (highlighted in bold above) and use it to query Elas
 ```
 curl -XGET http://0.0.0.0:32871
 ```
+
+## Configuration updates when using ElasticSearch on Local Environment
+
+ElasticSearch uses more memory within the local server environment, which can also cause services to stop working,n particularly when using Ubuntu. Altis includes an environment variable which can change the default memory limit for ElasticSearch called `ES_MEM_LIMIT`. 
```

Comment:
```suggestion
ElasticSearch requires more memory on certain operating systems such as Ubuntu or in Continuous Integration services. When ElasticSearch does not have enough memory it can cause other services to stop working. The Local Server supports an environment variable which can change the default memory limit for ElasticSearch called `ES_MEM_LIMIT`. 
```

Reply:
instead of "in Continuous Integration services" I think it should be "when using ..." as it's not actually "in" those services but using or utilizing them.

Reply:
@katmoody yep - I think I used that term as we'd be running elasticsearch within the server that the CI service gives us but you're right. You can edit my suggestion here then commit it